src/main/scala/gestalt/dotty/DottyToolbox.scala

```diff
@@ -524,8 +524,12 @@ class Toolbox(enclosingPosition: Position)(implicit ctx: Context) extends Tbox {
   }
 
   object Case extends CaseImpl {
+    private def ensureBlock(t:TermTree):TermTree = t match {
+      case unchanged:d.Block => unchanged
+      case other => d.Block(Nil,other)
+    }
```

Comment:
This would match the AST produced by the parser with the one produced by the quasiquotes. Later, however, I found out it was not necessary. Now it is properly removed.